.changeset/eighty-planets-provide.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
Any reason not to write a changeset? Updating internal dependencies also warrants a patch

Reply:
No reason, updating, good catch thank you @LekoArts!